src/main/java/com/klarna/hiverunner/StandaloneHiveServerContext.java

```diff
@@ -119,7 +167,22 @@ protected void configureSupportConcurrency(HiveConf conf) {
         hiveConf.setBoolVar(HIVE_SUPPORT_CONCURRENCY, false);
     }
 
-    protected void configureMetaStoreValidation(HiveConf conf) {
+    protected void configureMetaStore(HiveConf conf) {
+
+        try {
+            Class.forName(JDBCDriver.class.getName());
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+
+        // Set the hsqldb driver
+        metaStorageUrl = "jdbc:hsqldb:mem:" + UUID.randomUUID().toString();
+        hiveConf.set("datanucleus.connectiondrivername", "org.hsqldb.jdbc.JDBCDriver");
```

Comment:
Why use the JDBCDriver.class.getName() in one place and a string constant in these two places. Stick to one way. I prefer the class.getName() myself.